src/liquid/components/ld-toggle/ld-toggle.tsx

```diff
@@ -57,21 +75,39 @@ export class LdToggle implements InnerFocusable {
   @Watch('required')
   @Watch('value')
   updateHiddenInput() {
+    const outerForm = this.el.closest('form')
+    if (!this.hiddenInput && this.name && (outerForm || this.form)) {
+      this.hiddenInput = document.createElement('input')
```

Comment:
Same as for checkbox/radio.

```suggestion
      this.hiddenInput = document.createElement('input')
      this.hiddenInput.type = 'checkbox'
      this.hiddenInput.style.visibility = 'hidden'
      this.hiddenInput.style.position = 'absolute'
      this.hiddenInput.style.pointerEvents = 'none'
```